_includes/footer-idr.html

```diff
@@ -1,16 +1,16 @@
         <div class="callout large secondary">
             <div class="row">
                 <div class="row small-up-2 medium-up-5 large-up-5">
-                    <div class="column"><a href="http://www.openmicroscopy.org/" target="_blank"><img class="thumbnail their-logo" src="{{ "/img/logos/ome-logo-200.png" | prepend: site.baseurl }}?{{ site.time | date: '%s%N' }}" alt="OME"></a></div>
+                    <div class="column"><a href="https://www.openmicroscopy.org/" target="_blank"><img class="thumbnail their-logo" src="{{ "/img/logos/ome-logo-200.png" | prepend: site.baseurl }}?{{ site.time | date: '%s%N' }}" alt="OME"></a></div>
                     <div class="column"><a href="http://www.eurobioimaging.eu/" target="_blank"><img class="thumbnail their-logo" src="{{ "/img/logos/eurobioimaging_logo.gif" | prepend: site.baseurl }}?{{ site.time | date: '%s%N' }}" alt="Euro-Bioimaging"></a></div>
                     <div class="column"><a href="http://www.eurobioimaging.eu/" target="_blank"><img class="thumbnail their-logo" src="{{ "/img/logos/globalbioimaging_logo.png" | prepend: site.baseurl }}?{{ site.time | date: '%s%N' }}" alt="Global-Bioimaging"></a></div>
```

Comment:
Just noticed, should global-bioimaging link to http://www.eurobioimaging.eu/global-bioimaging instead of the top level?

Reply:
I'd defer to @jrswedlow

Reply:
At the moment, yes, that is the correct link.  You can shoot the messenger, but it won't change anything.